frontend/src/components/InputText.css

```diff
@@ -0,0 +1,52 @@
+.todoListMain .header input {
+    padding: 10px;
+    font-size: 16px;
+    border: 2px solid #FFF;
+    width: 165px;
+  }
+  .todoListMain .header button {
```

Comment:
I think there should be a space between line 6 and 7, i.e separate descriptions, to improve readability and formatting. Same goes for the rest of the code